src/ui/server/content-strings/fields/insurance/policy/index.ts

```diff
@@ -251,6 +251,7 @@ export const POLICY_FIELDS = {
   LOSS_PAYEE_DETAILS: {
     [LOSS_PAYEE_NAME]: {
       LABEL: 'Name of the loss payee',
+      MAXIMUM: 100,
```

Comment:
This could be in the new `constants/validation`, not a blocker since all name field validation rules will be re-visited soon, can be done then